modules/rest-util/src/blaze/middleware/output.clj

```diff
@@ -1,5 +1,8 @@
 (ns blaze.middleware.output
-  "JSON serialization middleware."
+  "Non-FHIR serialization middleware.
```

Comment:
BTW: @alexanderkiel would you have any suggestion for a better name for this?
"Non-FHIR" seems too general...

Reply:
Non-FHIR is ok. I use this middleware for the Admin-API were I return Non-FHIR JSON.